experts-migration-commands.php

```diff
@@ -0,0 +1,742 @@
+
+				$languages = '';
+				if ( $profile_meta['clone_other_languages'] ) {
+					$languages_heading = '<!-- wp:heading {"level":3} -->
+					<h3>Languages</h3><!-- /wp:heading -->';
+					foreach( $profile_meta['clone_other_languages'] as $language ){
+						if ( $language['text'] != '') {
+							$languages .= '<!-- wp:paragraph --><p>'.$language['text'].'</p><!-- /wp:paragraph -->';
+						}
+					}
+					if ( $languages && $languages != '' ){
+						$languages = $languages_heading . $languages;
+					} 
+					
+				}
+				$news_feed = '';
+				if ( $profile_meta['clone_news_feed']['textarea'] ) {
+					$news_feed = '<!-- wp:heading {"level":3} -->
+					<h3>In the Media</h3>
+					<!-- /wp:heading --><!-- wp:html -->'.$profile_meta['clone_news_feed']['textarea'].'<!-- /wp:html -->';
+					
+				}
+				$media_gallery = '';
+				if ( $profile_meta['clone_wordpress_gallery']['textarea'] ) {
+					$media_gallery = '<!-- wp:heading {"level":3} -->
+					<h3>Image Gallery</h3>
+					<!-- /wp:heading --><!-- wp:shortcode -->'.$profile_meta['clone_wordpress_gallery']['textarea'].'<!-- /wp:shortcode -->';
+					
+				}
+	
+				$embeds = '';
+				
+				if ( $profile_meta['clone_media_embed']['textarea'] ) {
+	
+					preg_match_all('/(?:youtu\.be\/|youtube\.com(?:\/embed\/|\/v\/|\/watch\?v=|\/user\/\S+|\/ytscreeningroom\?v=))([\w\-]{10,12})\b/', $profile_meta['clone_media_embed']['textarea'], $match);
+	
+					if ( ! empty( $match[1] ) ){
+						$embeds = '<!-- wp:heading {"level":3} -->
+						<h3>Videos</h3><!-- /wp:heading -->';
+						foreach( $match[1] as $id ){
+							$embeds .= '<!-- wp:embed {"url":"https://youtu.be/'.$id.'","type":"video","providerNameSlug":"youtube","responsive":true,"className":"wp-embed-aspect-16-9 wp-has-aspect-ratio"} -->
+			<figure class="wp-block-embed is-type-video is-provider-youtube wp-block-embed-youtube wp-embed-aspect-16-9 wp-has-aspect-ratio"><div class="wp-block-embed__wrapper">
+			https://youtu.be/'.$id.'
+			</div></figure>
+			<!-- /wp:embed --> ';
+						}
+					}					
+				}
+	
+				$formatted_content = sprintf( $gutenberg_template, 
+					$profile_pic_url, 
+					$name, 
+					$positions, 
+					$faculty_links,
+					$department_list, 
+					$campus_links,
+					$field_links,
+					$expertise,
+					$pronouns,
+					$pronunciation,
+					implode('', $email_list),
+					implode('', $phone_list),
+					implode('', $website_list),
+					implode('', $social_media_list),
+					$media_contact,
+					$languages,
+					$news_feed,
+					$media_gallery,
+					$embeds
+				);
+	
+				$maybe_update =  get_post_meta( $post_id, 'related_post', true );
+				$id = ( $maybe_update ? $maybe_update : '' );
+	
+				$excerpt = "<h3>$positions</h3>$expertise</p>";
+				$expert_post = array(
+					'ID'			=> $id,
+					'post_title'	=> $name,
+					'post_content'  => $formatted_content,
+					'post_status'   => 'publish',
+					'post_excerpt'		=> $excerpt
+	
+				);
+				$new_post_id = wp_insert_post( $expert_post, false );
```

Comment:
Don't forget to catch the other outcome of `wp_insert_post()` here where it doesn't return a Post ID otherwise the stuff that happens after this will fail loudly.